Tools/Scripts/libraries/webkitcorepy/webkitcorepy/testing/python_test_runner.py

```diff
@@ -82,6 +82,7 @@ def tests(self, args=None):
             if count:
                 continue
             sys.stderr.write("No tests match '{}'\n".format(pattern.replace('.*', '*').replace('**', '*')))
+            sys.stderr.write("Specify tests names in the form '<module>.<file>.<class>.<function>'\n")
```

Comment:
Better to print a sample usage, e.g.: `run-tests ews-build`

Reply:
Let me see if we can mechanically derive an example (note that this bit of code isn't specific to `Tools/CISupport/run-tests`, so our solution needs to be generic)